tests/chainer_tests/functions_tests/loss_tests/test_ctc.py

```diff
@@ -141,16 +141,6 @@ def setUp(self):
         self.blank_symbol = 3
 
 
-class TestCTCUseVolatile(unittest.TestCase):
-
-    def test_volatile(self):
-        xs_data = numpy.random.uniform(-1, 1, (4, 2, 3)).astype(numpy.float32)
-        t_data = numpy.array([[0, 1], [1, 0]]).astype(numpy.int32)
-        x = [chainer.Variable(x_data, volatile=True) for x_data in xs_data]
-        t = chainer.Variable(t_data, volatile=True)
-        functions.connectionist_temporal_classification(x, t, 2)
-
-
```

Comment:
I think it would be better to keep this unit test and to change `volatile` option with `no_backprop_mode()`.